pages/wiki/ip-connection.md

```diff
@@ -26,7 +26,11 @@ connmanctl&gt; connect wifi_CODE-FOR-YOUR-SSID
 connmanctl&gt; quit
 </code></pre>
 
-Verfify that you got an ip address asigned with <code># ip a show dev wlan0</code>.
+Check whether an IP address has been assigned to the watch using <code># ip a show dev wlan0</code> or <code>ifconfig wlan0</code>.
```

Comment:
For consistency either remove the # in the first command or add it to the other commands.

With that fixed please feel free to merge the PR yourself

Reply:
Thanks for taking the time to review!